node/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hydradx"
-version = "12.3.0"
+version = "13.1.0"
```

Comment:
is there any specific reason we start with 13.1, instead of 13.0?

Reply:
The PR we merged, related to genesis builder